src/tutorials/app.md

```diff
@@ -91,11 +91,11 @@ Password is `cozy`. In this mode HMR (Hot Module Replacement) is available.
 
 ## Going Further
 
-### What is `cozy.tools` ?
+### What is `cozy.localhost` ?
 
-You may see in our documentation that we use `*.cozy.tools` as domain for the local Cozy. Since Cozy uses subdomains to serve the applications, we can't use directly `localhost` as it.
+You may see in our documentation that we use `*.cozy.localhost` as domain for the local Cozy. Since Cozy uses subdomains to serve the applications, we can't use directly `localhost` as it.
```

Comment:
This paragraph was about `cozy.tools` vs `localhost`, now it seems more confusing as it is `cozy.localhost` vs `localhost`. Maybe the entire section can be removed?